cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -54,23 +54,23 @@ def new_compute_rate(enable_candu):
 @pytest.fixture(scope="module")
 def assign_chargeback_rate(new_compute_rate):
     """Assign custom Compute rate to the Enterprise and then queue the Chargeback report."""
-    enterprise = cb.Assign(
-        assign_to="The Enterprise",
-        selections={
-            'Enterprise': {'Rate': new_compute_rate}
-        })
-    enterprise.computeassign()
-    enterprise.storageassign()
+    for klass in (cb.ComputeAssign, cb.StorageAssign):
```

Comment:
Consider creating a singular fixture for this somewhere in `cfme/fixtures/` to reduce the code duplication throughout these tests. Might need some tweaks to the `'Default'` test case with a simple fixture providing that string for test parametrization.